index.js

```diff
@@ -1,9 +1,18 @@
 'use strict';
 
 const plugin = require('../..');
-const pluginName = require('../../package.json').name.replace(/^eslint-plugin-/, '');
 const selfPlugin = Object.assign({}, plugin);
 
+const pkgName = require('../../package.json').name;
+let pluginName;
+if (pkgName[0] === "@") {
+  const parts = pkgName.split('/');
```

Comment:
Should this `parts` variable be removed? It doesn't seem to be used anywhere.

Reply:
Yeah I've changed it to use a regex and forgot to remove the split